zerotier: перезапуск при изменении /etc/config/zerotier

Без service_triggers «Сохранить и применить» в LuCI (luci-app-zerotier)
не перезапускало службу.

--- a/net/zerotier/files/etc/init.d/zerotier
+++ b/net/zerotier/files/etc/init.d/zerotier
@@ -109,3 +109,13 @@
 	config_foreach stop_instance 'zerotier'
 	rm -f ${CONFIG_PATH}
 }
+
+# ax50-openwrt: применение настроек из LuCI (procd reload trigger)
+reload_service() {
+	stop
+	start
+}
+
+service_triggers() {
+	procd_add_reload_trigger 'zerotier'
+}
